packages/css/src/components/alert/alert.scss

```diff
@@ -21,38 +21,25 @@
   flex: auto;
 }
 
-@mixin reset {
-  -webkit-text-size-adjust: 100%;
-}
-
-.amsterdam-alert__title {
-  color: var(--amsterdam-alert-title-color);
-  font-family: var(--amsterdam-alert-title-font-family);
-  font-size: var(--amsterdam-alert-title-spacious-font-size);
-  font-weight: var(--amsterdam-alert-title-font-weight);
-  line-height: var(--amsterdam-alert-title-spacious-line-height);
-
-  .amsterdam-theme--compact & {
-    font-size: var(--amsterdam-alert-title-compact-font-size);
-    line-height: var(--amsterdam-alert-title-compact-line-height);
-  }
-
-  @include reset;
-}
-
 .amsterdam-alert--error {
-  background-color: var(--amsterdam-alert-error-background-color);
   border-color: var(--amsterdam-alert-error-border-color);
 }
 
+.amsterdam-alert--info {
+  border-color: var(--amsterdam-alert-info-border-color);
+}
+
 .amsterdam-alert--success {
-  background-color: var(--amsterdam-alert-success-background-color);
   border-color: var(--amsterdam-alert-success-border-color);
 }
 
+.amsterdam-alert--warning {
+  border-color: var(--amsterdam-alert-warning-border-color);
+}
+
 /* todo: move to action button component */
 .amsterdam-alert__close {
-  background-color: var(--amsterdam-alert-close-background-color);
+  background-color: var(--amsterdam-alert-close-button-background-color);
```

Comment:
This is a wasted effort, we should wait for the IconButton PR and then use that component. @alimpens has left out Alert on purpose.